jest.config.js

```diff
@@ -12,4 +12,6 @@ module.exports = {
     'jest-watch-typeahead/filename',
     'jest-watch-typeahead/testname',
   ],
+  collectCoverage: false,
```

Comment:
why is coverage set up to be collected but set to false?

Reply:
We don't have anywhere to collect coverage yet, so I thought I'd enable it in a follow-up PR.